src/AutoVizuA11y.tsx

```diff
@@ -51,6 +60,7 @@ type AutoVizuA11yProps = {
 	title: string;
```

Comment:
don't forget to export this type so that it can be consumed outside.
Also, the comments you have for each prop as jsdoc, they can be moved to each property in this type, which makes the file a bit cleaner and still documented.

Watch out for indentation on the jsdoc examples :) It seems like the example has a few tabs too many :)

Reply:
I believe to have changed it as you suggested. Even then when i hover `AutoVizuA11y` i dont see the comments for each individual param :/